main.cpp

```diff
@@ -275,296 +300,283 @@ struct Bakery
+{
+    if (hTotalFloorSize / (numPeople+numAnimals) > 20)
+        return true;
+    else
+        return false;
+}
 
 struct VCA
 {
-    //minimum gain (double)
     double minGain = -100.66;
-    //maximum gain (double)
     double maxGain = 10.5;
-    //initial gain (double)
     double initGain = -20.1;
-    //modulation attenuator (float)
     float modAtt = -56.2f;
-    //inputs (int)
     int inputs = 2;
 
-    //set inital gain
-    float setInitGain(double minGain, double maxGain);
-    //add rhythm to signal
-    void modulateOutput(float modAtt, float modSpeed);
-    //silence the signal
+    double setInitGain(double vMinGain, double vMaxGain);
+    void modulateOutput(float vModAtt, float modSpeed);
     void silenceOutput();
 };
 
-/*
-Thing 6) filter
-5 properties:
-    1) cutoff frequency (float)
-    2) resonance (float)
-    3) filter type (std::string)
-    4) input gain (double)
-    5) overdrive (float)
-3 things it can do:
-    1) cut low frequencies
-    2) cut high frequencies
-    3) self-resonate
- */
+double VCA::setInitGain(double vMinGain, double vMaxGain)
+{
+    std::cout << "Setting init gain. Current minimum is: " << vMinGain << " and maximum is: " << vMaxGain << std::endl;
+
+    return (vMinGain-vMaxGain);
+}
+
+void VCA::modulateOutput(float vModAtt, float modSpeed)
+{
+    std::cout << "Modulating output by: " << vModAtt << " with the frequency of: " << modSpeed << std::endl;
+}
+
+void VCA::silenceOutput()
+{
+}
 
 struct Filter
 {
-    //1) cutoff frequency (float)
     float cutoffFreq = 13200.2f;
-    //2) resonance (float)
     float resonance = 0.77f;
-    //3) filter type (std::string)
     std::string filterType = "Low Pass";
-    //4) input gain (double)
     double inputGain = -5.3;
-    //5) overdrive (float)
     float overdriveAmount = 1.7f;
 
-    //1) cut low frequencies
     void cutLows(float cutoffFreq, float resonance);
-    //2) cut high frequencies
     void cutHighs(float cutoffFreq, float resonance);
-    //3) self-resonate
-    bool selfResonate(float resonance, double inputGain);
+    bool selfResonate(float fResonance, double fInputGain);
 };
 
-/*
-Thing 7) envelope
-5 properties:
-    1) attack time (float)
-    2) decay time (float)
-    3) sustain level (double)
-    4) release time (float)
-    5) curve (float)
-3 things it can do:
-    1) create long sustained signals
-    2) create short staccato signals
-    3) loop over time
- */
+void Filter::cutLows(float fCutoffFreq, float fResonance)
+{
+    std::cout << "Cutting low frequencies below: " << fCutoffFreq << "with a Q of: " << fResonance << std::endl;
+}
+void Filter::cutHighs(float fCutoffFreq, float fResonance)
+{
+    std::cout << "Cutting high frequencies above: " << fCutoffFreq << "with a Q of: " << fResonance << std::endl;
+}
+bool Filter::selfResonate(float fResonance, double fInputGain)
+{
+    if (fResonance > 0.9f && fInputGain > -35.0)
+        return true;
+    else
+        return false;
+}
```

Comment:
you can truncate this to:
```
return fResonance > 0.9f && fInputGain > -35.0;
```
There is no need for `if/else`
- [ ] revise

Reply:
This is great, I love when code can be shortened this much!